src/apis/nimbus.js

```diff
@@ -3,13 +3,28 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-/* eslint-disable-next-line no-unused-vars */
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "nimbus" }]*/
+const { ExperimentManager } = ChromeUtils.importESModule("resource://nimbus/lib/ExperimentManager.sys.mjs");
+
 var nimbus = class extends ExtensionAPI {
-  getAPI() {
+  getAPI(context) {
     return {
       experiments: {
-        nimbus: {},
+        nimbus: {
+          async enrollExperiment(jsonData) {
+            try {
+              const source = "nimbus-devtools";
+              const recipe = JSON.parse(jsonData);
+              const result = await ExperimentManager.enroll(recipe, source);
+              return result;
```

Comment:
We don't need the actual enrollment returned. We should consider just returning whether or not the enrollmetn was successful (i.e., result !== null)